site/en/guide/autodiff.ipynb

```diff
@@ -947,11 +947,11 @@
       "source": [
         "Some `tf.Operation`s are **registered as being non-differentiable** and will return `None`. Others have **no gradient registered**.\n",
         "\n",
-        "The [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops) page shows which low-level ops have gradients registered.\n",
```

Comment:
Please don't remove links as someone can refer from them and it will be easy for others to find the documentation of key terms.

Reply:
Hi, same as in https://github.com/tensorflow/docs/pull/1736#discussion_r518244265 - I thought these get auto-linked to API references during the notebook-to-TensorFlow.org preprocessing